x-pack/plugins/cases/common/types/domain/configure/v1.test.ts

```diff
@@ -193,6 +194,7 @@ describe('configure', () => {
       label: 'Text Custom Field',
       type: CustomFieldTypes.TEXT,
       required: true,
+      defaultValue: 'foobar',
```

Comment:
I think the `defaultRequest` should be with `required: true`. This way we can test the other path (`required: false`). On the dedicated test for the default value, we can set `required: true` and test it. Same for `TextCustomFieldConfigurationRt`. Wdyt?

Reply:
> I think the defaultRequest should be with required: true. 

I'm a bit confused, isn't it already? (line 196)

Reply:
Sorry, my mistake. I mean it should be `required: false`.